pkg/cmd/container/create.go

```diff
@@ -340,6 +341,13 @@ func generateRootfsOpts(args []string, id string, ensured *imgutil.EnsuredImage,
 		opts = append(opts, oci.WithRootFSPath(absRootfs), oci.WithDefaultPathEnv)
 	}
 
+	entrypointPath := ""
+	if len(ensured.ImageConfig.Entrypoint) > 0 {
+		entrypointPath = ensured.ImageConfig.Entrypoint[0]
+	} else if len(ensured.ImageConfig.Cmd) > 0 {
+		entrypointPath = ensured.ImageConfig.Cmd[0]
+	}
+
```

Comment:
There might be an easier way to determine the entrypoint executable path, if there is I am open to updating